app/views/admin/cms/collections/_form.html.erb

```diff
@@ -1,5 +1,7 @@
-<%= form_for [:admin, :cms, @resource] do |f| %>
-  <%= render "admin/cms/shared/errors" %>
+<%= form_for [:admin, :cms, @collection] do |f| %>
```

Comment:
Uploads still work without it, from reading the documentation it only appears to be necessary for multiple file uploads and it's added to the file_field.
https://github.com/carrierwaveuploader/carrierwave#multiple-file-uploads